packages/connect-migrate/src/migrations/v0.13.1.ts

```diff
@@ -0,0 +1,218 @@
+// Copyright 2021-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Scanned } from "../lib/scan";
+import { Logger, PrintFn } from "../lib/logger";
+import { CommandLineArgs } from "../arguments";
+import modifyImports from "./v0.13.1-transform";
+import { MigrateError, MigrateSuccess, Migration } from "../migration";
+import { runInstall } from "../lib/run";
+import { writePackageJsonFile } from "../lib/package-json";
+import {
+  DependencyReplacement,
+  replaceDependencies,
+} from "../lib/replace-dependencies";
+import { updateSourceFile } from "../lib/update-source-file";
+import { migrateSourceFiles } from "../lib/migrate-source-files";
+import { migratePackages } from "../lib/migrate-packages";
+import { migrateLockFiles } from "../lib/migrate-lock-files";
+
+/**
+ * The latest available version of protoc-gen-connect-web is v0.11.0.
+ * v0.12.0 removed it. The package was never moved to the connectrpc org.
+ *
+ * Since this command cannot modify the project completely to switch
+ * to protoc-gen-connect-es, it will abort the migration unless the
+ * --force flag is set.
+ */
+const oldPluginReplacement = {
+  from: { name: "@bufbuild/protoc-gen-connect-web", range: "*" },
+  to: { name: "@connectrpc/protoc-gen-connect-es", version: "0.13.1" },
+} satisfies DependencyReplacement;
+
+/**
+ * v0.9.0 dropped several exports from @bufbuild/connect-web that had
+ * been moved to a shared package in v0.8.0.
+ *
+ * Since this command cannot currently modify the imports accordingly,
+ * it will abort the migration unless the --force flag is set.
+ */
+const removedWebExportReplacement: DependencyReplacement = {
+  from: { name: "@bufbuild/connect-web", range: "<0.9.0" },
+  to: { name: "@connectrpc/connect-web", version: "0.13.1" },
+};
```

Comment:
Same here - the migration will be aborted. If necessary, we can update the codemod to handle this.